components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -132,100 +144,145 @@ export default function ProjectSettingsView() {
                     Update Name
                 </Button>
             </form>
-            <Heading2 className="mt-12">Prebuilds</Heading2>
-            <Subheading>Choose the workspace machine type for your prebuilds.</Subheading>
-            <div className="max-w-md">
-                <SelectWorkspaceClassComponent
-                    selectedWorkspaceClass={project.settings?.workspaceClasses?.prebuild}
-                    onSelectionChange={setWorkspaceClassForPrebuild}
+            <div>
+                <Heading2 className="mt-12">Prebuilds</Heading2>
+                <CheckboxInputField
+                    label="Enable prebuilds"
+                    hint={
+                        enablePrebuilds ? (
+                            <span>
+                                Make sure to push a <code>.gitpod.yml</code> file to the repository.{" "}
```

Comment:
**nipick**: Minor rephrase for brevity and explicity.

```suggestion
                                Prebuilds will run for any <code>before</code> or <code>init</code> tasks.{" "}
```

<img width="950" alt="examples" src="https://github.com/gitpod-io/gitpod/assets/120486/95a9d185-0763-4c76-9265-c52798c553fc">